WordPress/Classes/Utility/SharedStrings.swift

```diff
@@ -10,6 +10,7 @@ enum SharedStrings {
         static let edit = NSLocalizedString("shared.button.edit", value: "Edit", comment: "A shared button title used in different contexts")
         static let add = NSLocalizedString("shared.button.add", value: "Add", comment: "A shared button title used in different contexts")
         static let remove = NSLocalizedString("shared.button.remove", value: "Remove", comment: "A shared button title used in different contexts")
+        static let delete = NSLocalizedString("shared.button.remove", value: "Delete", comment: "A shared button title used in different contexts")
```

Comment:
I ended up not using it, but it's a good candidate for a standard button.

Reply:
Should the key be `shared.button.delete`?

Reply:
Good catch, fixed.